app/models/zone.rb

```diff
@@ -48,6 +52,9 @@ def find_master_server
   end
 
   def self.seed
+    create_with(:description => "Maintenance Zone", :visible => false).find_or_create_by!(:name => 'maintenance') do |_z|
```

Comment:
Probably not _that_ big of a concern but if someone did have a zone named `'maintenance'` this would hijack that, maybe something like `'__maintenance'`...that wouldn't look nice on the EMS's summary page though

Reply:
That's a good point...I'm not sure how we should handle that possiblity...Maybe rename their existing maintenance zone?

Reply:
hm, I agree

Reply:
I think the best place for renaming is data migration, there is not possible in `Zone.seed`  to say if it's our zone or user zone.
https://github.com/ManageIQ/manageiq-schema/pull/222/files#diff-bbb44967ce5928b07e37c559a29f3785R10